### PR TITLE
LPS-169269 Improve the semantics of the search input

### PR DESCRIPTION
**Pull request by veroglez**
It fixes:
- https://issues.liferay.com/browse/LPS-169269
- https://issues.liferay.com/browse/LPS-169264

### CODE REVIEW

**Comment by eltonccdantas** [comment]
All green, great!

File: modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/management_toolbar/SearchControls.js [hunk 1]
```diff
@@ -40,13 +40,14 @@ const SearchControls = ({
 				<ClayInput.Group>
 					<ClayInput.GroupItem>
 						<ClayInput
+							aria-label={`${Liferay.Language.get('search')}:`}
```
Comment (markocikos) [inline modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/management_toolbar/SearchControls.js:43]: Does the colon have any significance? :thinking: Is there a difference for a screen reader between this and just 
```html
aria-label={Liferay.Language.get('search')}
```

**Review by markocikos — COMMENTED** [review]
@veroglez Please see question inline. Other than that, looks good!

File: modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/management_toolbar/SearchControls.js (hunk 1 above)
Comment (veroglez) [reply modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/management_toolbar/SearchControls.js:43]: Hi @markocikos! Yes, screen readers detect punctuation symbols and each user can have a specific setting for that.

**Review by markocikos — APPROVED** [review]

**Comment by markocikos** [comment]
ci:forward

**Pull request closed without merging**